draft-ietf-tls-tls13.md

```diff
@@ -3372,8 +3372,7 @@ that are not known to be supported by the client. This fallback chain MAY
 use the deprecated SHA-1 hash algorithm only if the "signature_algorithms"
 extension provided by the client permits it.
 If the client cannot construct an acceptable chain using the provided
-certificates and decides to abort the handshake, then it MUST abort the
-handshake with an "unsupported_certificate" alert.
+certificates then it MUST abort the handshake.
```

Comment:
I'd like to keep a specific alert in as default, so I suggest something like this instead:

"If the client cannot construct an acceptable chain using the provided certificates and decides to abort the handshake, then it MUST abort the handshake with an appropriate certificate-related alert. (by default, "unsupported_certificate"; see {{error-alerts}} for more)"